openvoxserver/Containerfile

```
@@ -81,19 +80,15 @@ COPY --from=builder /usr/local/bin/r10k /usr/local/bin/
 
 ADD https://apt.overlookinfratech.com/openvox${OPENVOX_RELEASE}-release-ubuntu${UBUNTU_VERSION}.deb /
 RUN apt-get update && \
-    apt-get install -y ca-certificates && \
-    dpkg -i /openvox${OPENVOX_RELEASE}-release-ubuntu${UBUNTU_VERSION}.deb && \
+    apt-get install -y ca-certificates /openvox${OPENVOX_RELEASE}-release-ubuntu${UBUNTU_VERSION}.deb && \
     rm /openvox${OPENVOX_RELEASE}-release-ubuntu${UBUNTU_VERSION}.deb
 
 RUN groupadd -g ${OPENVOX_USER_GID} puppet && \
     useradd -m -u ${OPENVOX_USER_UID} -g puppet puppet && \
     chmod +x /docker-entrypoint.sh /healthcheck.sh /docker-entrypoint.d/*.sh && \
     apt-get update && \
     apt-get upgrade -y && \
-    apt-get install -y $PACKAGES && \
-    apt-get install -y openvox-agent=${OPENVOXAGENT_VERSION}-1+ubuntu${UBUNTU_VERSION} && \
-    apt-get install -y openvox-server=${OPENVOXSERVER_VERSION}-1+ubuntu${UBUNTU_VERSION} && \
-    apt-get install -y openvoxdb-termini=${OPENVOXDB_VERSION}-1+ubuntu${UBUNTU_VERSION} && \
+    apt-get install -y $PACKAGES openvox-server=${OPENVOXSERVER_VERSION}-1+ubuntu${UBUNTU_VERSION} && \
```

As noted elsewhere, I think this should work:
```suggestion
    apt-get install -y $PACKAGES openvox-server=${OPENVOXSERVER_VERSION}-* && \
```